scenarios/kubernetes_build.py

```diff
@@ -120,8 +120,6 @@ def main(args):
         push_build_args.append('--bucket=%s' % args.release)
     if args.registry:
         push_build_args.append('--docker-registry=%s' % args.registry)
-    if args.hyperkube:
-        env['KUBE_BUILD_HYPERKUBE'] = 'y'
```

Comment:
are we using this on older branches?

Reply:
the CI jobs didn't seem to need it when i eye-balled them.